packages/app-data/src/schemas/v1.6.0.json

```diff
@@ -0,0 +1,84 @@
+{
+  "$id": "https://cowswap.exchange/schemas/app-data/v1.5.0.json",
```

Comment:
I know it's not from here, but the schema has a bug: should be 1.6.0

Reply:
Fixed